docs/modules/ROOT/pages/advanced_usage/configuration_file.adoc

```diff
@@ -1,13 +1,24 @@
 = Configuration File
+:toc:
+:ini-file-format-url: https://en.wikipedia.org/wiki/INI_file
 
 The ownCloud Client reads a configuration file.
 You can locate this configuration file as follows:
 
-On Linux distributions: `$HOME/.config/ownCloud/owncloud.cfg`
-On Microsoft Windows systems: `%APPDATA%\ownCloud\owncloud.cfg`
-On MAC OS X systems: `$HOME/Library/Preferences/ownCloud/owncloud.cfg`
+[cols="25%,75%",options="header"]
+|===
+| System
+| Location
+
+| *Linux* 
+| `$HOME/.config/ownCloud/owncloud.cfg`
+| *Microsoft Windows* 
+| `%APPDATA%\ownCloud\owncloud.cfg`
+| *macOS X* 
```

Comment:
I think it's either `macOS`  without `X` or `(Mac) OS X` 

@guruz @michaelstingl correct me if I'm wrong here

Reply:
`macOS` [it is](https://en.wikipedia.org/wiki/MacOS)!

Reply:
Fixed.